sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala

```diff
@@ -84,6 +85,21 @@ class TextFileFormat extends FileFormat with DataSourceRegister {
     }
   }
 
+  override def canSplitFiles(files: Seq[Path], conf: Configuration): Boolean = try {
```

Comment:
@rxin 

By inlining `TextInputFormat.isSplitable`,  do you mean copying the code in FileInputFormat.isSplittable and put it here?

```
class FileInputFormat {
... 
 protected boolean isSplitable(FileSystem fs, Path file) {
    final CompressionCodec codec = compressionCodecs.getCodec(file);
    if (null == codec) {
      return true;
    }
    return codec instanceof SplittableCompressionCodec;
  }
}
```